extensions/emmet/src/abbreviationActions.ts

```diff
@@ -53,7 +53,11 @@ export function wrapWithAbbreviation(args: any) {
 
 		editor.selections.forEach(selection => {
 			let rangeToReplace: vscode.Range = selection.isReversed ? new vscode.Range(selection.active, selection.anchor) : selection;
-			if (rangeToReplace.isEmpty) {
+			if (!rangeToReplace.isSingleLine && rangeToReplace.end.character === 0) {
```

Comment:
we should do something similar to the `Emmet: Wrap individual lines with abbreviation` feature as well